### DISCUSSION TITLE
I had pay fee and got license code but request code again !

### DISCUSSION THREAD

**Discussion by arthur5537** [aclap-dev/video-downloadhelper#1518] [Help and Q&A]
### Your question

Hi,
I had pay fee and got license code but request code again, can you check for me,
I am Arthurlee , arthurlee5537@gmail.com .......thanks .

### Addon & CoApp version and details

Hi,
I had pay fee and got license code but request code again, can you check for me,
I am Arthurlee , arthurlee5537@gmail.com .......thanks .

**Comment by jcfrog** [comment]
Hi, your license is still valid, you probably just need to revalidate it: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key

**Comment by arthur5537** [comment]
Thanks your help, I follow your instruction to check my key, but show  " Serial
number not set,
can you help me again ? thanks!


jcfrog ***@***.***> 於 2024年12月5日 週四 下午3:33寫道：

**Reply by jcfrog** [reply]
did you click on "Check license key" link?
if yes, do you get an error message?

**Comment by buditanrim** [comment]
hello, what's the email address that sent the license key? I need to find it. Thank you

**Reply by jcfrog** [reply]
hi, you can recover your license from here: https://www.downloadhelper.net/recover-license
if any problem please email us at licenses@downloadhelper.net

**Comment by arthur5537** [comment]
yes, I had click check license key, and get the

jcfrog ***@***.***> 於 2024年12月5日 週四 下午9:28寫道：

**Comment by arthur5537** [comment]
Hi,

thanks you help, but I meet a new issue, I download video can downloaded,
but just almost 255Mbytes /Mp4  is stop, I don't know why ?
can you help me ? thanks !    video 10.0.1.198.2V

Arthur
